src/test/scala/mockws/Helpers.scala

```diff
@@ -0,0 +1,13 @@
+package mockws
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import play.api.mvc.{DefaultActionBuilder, PlayBodyParsers}
+import scala.concurrent.ExecutionContext.Implicits._
+
+object Helpers {
+  implicit val as = ActorSystem()
+  implicit val materializer = ActorMaterializer()
```

Comment:
I am thinking of a clean way to deal with it. I might refer to Play's source code to see how they deal with it.
In the beginning I used Play's `play.api.test.NoMaterializer`, which doesn't allocate any resources but unfortunately it didn't work.